core/commonMain/src/kotlinx/serialization/internal/CollectionSerializers.kt

```diff
@@ -82,7 +81,7 @@ public sealed class ListLikeSerializer<Element, Collection, Builder>(
     }
 }
 
-@InternalSerializationApi // Not intended for public use at all
+@InternalSerializationApi
```

Comment:
Why not `@PublishedApi internal class` too?

Reply:
Used in ProtoBuf, I've left a comment